README.md

```diff
@@ -1,430 +1,100 @@
 # Stripe CLI
 
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/stripe/stripe-cli)
 [![Build Status](https://travis-ci.com/stripe/stripe-cli.svg?token=eQWDVpt1sJR63TxbC1KA&branch=master)](https://travis-ci.com/stripe/stripe-cli)
+![GitHub](https://img.shields.io/github/license/stripe/stripe-cli)
 
-_🏗 The Stripe CLI is currently in beta! We're working on more features to make the experience great. If you have any [feedback](https://stri.pe/cli-feedback), find [issues](https://github.com/stripe/stripe-cli/issues), or would like to be involved in more active testing, please let us know!_
-
-The Stripe CLI is a command-line interface for Stripe that can:
-
-1. `login` to your Stripe account and authenticate the CLI
-2. `listen` for webhooks and forward them to a local server
-3. Run resource commands for things like `stripe charges create`
-4. Run `get`, `post`, and `delete` commands to the Stripe API
-5. `trigger` a limited set of webhook events
-6. Tail your test mode API request logs
-7. Pull Stripe status from status.stripe.com
-
-The main focus for this initial release is to improve the developer experience while integrating and testing webhooks. Interactions through the CLI are currently limited to test mode only.
-
-## Table of Contents
-
-* [Stripe CLI](#stripe-cli)
-  * [Installation](#installation)
-    * [macOS](#macos)
-    * [Linux](#linux)
-    * [Windows](#windows)
-    * [Docker](#docker)
-  * [Commands](#commands)
-    * [login](#login)
-    * [listen](#listen)
-    * [Resource commands](#resource-commands)
-    * [get, post, and delete](#get-post-and-delete)
-    * [trigger](#trigger)
-    * [logs tail](#logs-tail)
-    * [samples](#samples)
-    * [status](#status)
-    * [config](#config)
-    * [open](#open)
-  * [Developing the Stripe CLI](#developing-the-stripe-cli)
-    * [Installation](#installation-1)
-    * [Linting](#linting)
-    * [Tests](#tests)
-    * [Releasing](#releasing)
-  * [License](#license)  
-    
+The Stripe Command-line interface (CLI) makes it easy for you to build, test and manage your Stripe integration right from the terminal by integrating seamlessly with your Stripe account.
```

Comment:
I kind of don't want to do the "command-line interface (CLI)" part since most people will likely know what a CLI is. We don't do that with API for the docs.

Reply:
"build, test, and manage your Stripe integration right from the command line"

Reply:
The intent here is to align on the value prop and wording used to describe the Stripe CLI, so it's consistent across blog post, stripe docs, and the readme which serves as the "landing page" for the Stripe CLI.

What's the specific proposal?
`The Stripe CLI makes it easy for you to build, test and manage your Stripe integration right from the terminal by integrating seamlessly with your Stripe account.`

Reply:
Wonder if we should hold off on this until we figure out how to better position the value proposition around the "manage" side

Reply:
Addressed